File: 프로그래머스/unrated/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        
        myString = myString.replaceAll("A","b");
        myString = myString.replaceAll("B","A");
        myString = myString.replaceAll("b","B");
        
        answer = myString.contains(pat) ? 1 : 0;
        
        return answer;
    }
}

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(const char* myString, const char* pat) {
    char tmp[strlen(myString)+1];
    
    for(int i=0;i<strlen(myString);i++){
        tmp[i] = myString[i] == 'A' ? 'B': 'A';
    }  
    return strstr(tmp, pat) ? 1 : 0;
}